lib/array_practice.rb

```diff
@@ -0,0 +1,51 @@
+array_of_integers = [*0...50]
+
+def all_odds(array)
+  # return all odd numbers in array_of_integers
+  array.select(&:odd?)
+end
+
+def all_evens(array)
+  # return all even numbers in array_of_integers
+  array.select(&:even?)
+end
+
+def all_squares(array)
```

Comment:
I would use "all_square_roots" and consistently use that language throughout

Reply:
I've added better comments and some definitions in the readme to make this clearer.